doc/suite_runner_api.md

```diff
@@ -47,7 +47,7 @@ runner:
   by default.
 * `attempts`: Maximum number of times to run a test; if >1 the test will be
   retried on failure. Defaults to 1.
-* `parallelism`: Number of tests to run in parallel. Defaults to 8.
+* `parallelism`: Number of tests to run in parallel. Defaults to `Math.floor(numberOfCpus * 1.5)`.
```

Comment:
I think os.cpus() is safely x-platform 

http://nodejs.org/api/os.html#os_os_cpus

> On Jan 19, 2015, at 2:35 PM, Per Eckerdal <notifications@github.com> wrote:
> 
> In doc/suite_runner_api.md:
> 
> > @@ -47,7 +47,7 @@ runner:
> >    by default.
> >  * `attempts`: Maximum number of times to run a test; if >1 the test will be
> >    retried on failure. Defaults to 1.
> > -* `parallelism`: Number of tests to run in parallel. Defaults to 8.
> > +* `parallelism`: Number of tests to run in parallel. Defaults to `Math.floor(numberOfCpus * 1.5)`.
> That would be good, but do you know of an easy cross platform way of getting the number of CPUs? I didn't know one off the top of my head, which was why I didn't do it.
> 
> —
> Reply to this email directly or view it on GitHub.
>